angular-notify.js

```diff
@@ -57,19 +66,26 @@ angular.module('cgNotify', []).factory('notify',['$timeout','$http','$compile','
                 angular.element(args.container).append(templateElement);
                 messageElements.push(templateElement);
 
-                if (args.position === 'center'){
-                    $timeout(function(){
-                        templateElement.css('margin-left','-' + (templateElement[0].offsetWidth /2) + 'px');
-                    });
+                switch (args.position){
+                    case 'center':  templateElement.addClass('cg-notify-message-center');
+                                    templateElement.css('margin-left','-' + (templateElement[0].offsetWidth /2) + 'px');
```

Comment:
Thanks, and thanks for this PR! I'm using it primarily because of the custom duration feature, but I certainly don't mind getting the other improvements for free as well :)